vpp-manager/uplink/avf.go

```diff
@@ -131,6 +131,15 @@ func (d *AVFDriver) RestoreLinux() {
 }
 
 func (d *AVFDriver) CreateMainVppInterface(vpp *vpplink.VppLink, vppPid int) error {
+	if d.pfPCI != "" {
+		/* We were passed a PF, move it to vpp's NS so it doesn't
+		   conflict with the VF */
```

Comment:
Isn't the conflict with the vpptap rather than the VF ?

Reply:
Right, typoed...